test/test_datasets_download.py

```diff
@@ -149,13 +149,13 @@ def assert_server_response_ok():
         raise AssertionError(str(error)) from error
 
 
-def assert_url_is_accessible(url, timeout=5.0):
+def assert_url_is_accessible(url, timeout=30.0):
```

Comment:
Is this the minimum value that will work? I'm just worried that this has the potential of increasing a lot the run-time of multiple unit-tests. Any other workaround?

Reply:
Nope. Tests are still failing.